utils/R/do.conversions.R

```diff
@@ -1,9 +1,8 @@
-##' @export
+##' @export do.conversions
```

Comment:
This change is just reflective of a "." changed to a "," automatically and then a "." after Istem advised me to delete an extra @export statement.  @isfer would know more about if do.conversions needs to be exported.